Content.Server/Destructible/Thresholds/Behaviors/SpawnEntitiesBehavior.cs

```diff
@@ -25,6 +19,10 @@ public sealed class SpawnEntitiesBehavior : IThresholdBehavior
         public void Execute(EntityUid owner, DestructibleSystem system)
         {
             var position = system.EntityManager.GetComponent<TransformComponent>(owner).MapPosition;
+            
+            var offsetPosition = () => position.Offset((
+                (system.Random.NextFloat() * 2 - 1) * Offset,
+                (system.Random.NextFloat() * 2 - 1) * Offset));
```

Comment:
Does NextFloat have the min and max support?

Reply:
also why not robustrandom

Reply:
It does. Switched to using that.